other/Perl Jam 2 - RCE.bcheck

```diff
@@ -0,0 +1,35 @@
+metadata:
+    language: v2-beta
+    name: "Perl JAM 2"
+    description: "Perl JAM 2 - RCE"
+    author: "Andrej Šimko @ Accenture"
+    tags: "Perl JAM 2","Perl","Remote Code Execution","RCE"
+
+define: 
+    request_all_headers = `{base.request.headers}`
+    multi_line = `{regex_replace({request_all_headers},"[\s\S]+boundary=","")}`
+    boundary = `{regex_replace({multi_line},"[\r\n].*","")}`
+    request_original_body = `{base.request.body}`
+
+    #this should potentially be changed to work on Windows deployment
```

Comment:
would this be better at the top of the define block to draw more attention to it? The user needs to potentially change this value, while the others are fixed.